cluster-autoscaler/cloudprovider/packet/packet_cloud_provider.go

```diff
@@ -34,8 +34,8 @@ import (
 )
 
 const (
-	// ProviderName is the cloud provider name for Packet
-	ProviderName = "packet"
+	// ProviderName is the cloud provider name for Equinix Metal
+	ProviderName = "equinix-metal"
```

Comment:
This would be a breaking change for users.

Reply:
I guess, this should not be a breaking change because this variable is used to return the `Name()` of the provider whereas ib builders of the cloud provider `buildCloudProvider` and `builder_all` its handled as both: packet and equinixmetal